File: proyectos_c#/EntityFramework/CRUDSystem/CRUDSystem/Models/Detail.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CRUDSystem.Models
{
    class Detail
    {
        [Key]
        public int ID { get; set; }
        public string Fname { get; set; }
        public string Lname { get; set;  }
        public int Age { get; set; }
        public string Address { get; set; }
        public DateTime DOB { get; set; }
    }
}

File: proyectos_basic/web/FormGridView/FormGridView/Models/Detail.vb
Imports System.ComponentModel.DataAnnotations
Imports System.Threading.Tasks
Namespace Models
    Public Class Detail

        Private _id As Integer
        Private _descripcion As String
        Private _precio As Decimal
        <Key>
        Public Property Id As Integer
            Get
                Return _id
            End Get
            Set(value As Integer)
                _id = value
            End Set
        End Property

        Public Property Descripcion As String
            Get
                Return _descripcion
            End Get
            Set(value As String)
                _descripcion = value
            End Set
        End Property

        Public Property Precio As Decimal
            Get
                Return _precio
            End Get
            Set(value As Decimal)
                _precio = value
            End Set
        End Property
    End Class

End Namespace
